tests/callbacks/fitting/test_fitting_methods.py

```diff
@@ -823,3 +824,145 @@ def test_guess_given_flat_data(self):
             outp = NegativeTrapezoid.guess()(x, y)
             # check that with flat data gradient guess is 0
             assert outp["gradient"] == pytest.approx(0.0, rel=1e-2)
+
+
+@pytest.fixture
+def values() -> dict[str, float]:
```

Comment:
Not sure these fixtures belong at the top-level of this file (especially with relatively ambiguous names like `x` or `values`). Do they need to be fixtures, or could they be constants within the `TestMuonMomentum` class?